tokio-trace/tokio-trace-core/src/subscriber.rs

```diff
@@ -442,4 +442,23 @@ impl Interest {
             _ => false,
         }
     }
+
+    /// More relevant documentation
+    pub fn and(self, rhs: Interest) -> Self {
```

Comment:
This should not be public
```suggestion
    pub(crate) fn and(self, rhs: Interest) -> Self {
```